templates/authentication.dot

```diff
@@ -1,2 +1,4 @@
-To perform this operation, you must be authenticated by means of one of the following methods:
```

Comment:
Doesn't this get converted to a <p> anyway as it's markdown?

Reply:
Good question. I'll try without